devenv/src/cli.rs

```diff
@@ -23,6 +23,9 @@ pub struct GlobalOptions {
     #[arg(short, long, global = true, help = "Enable debug log level.")]
     pub verbose: bool,
 
+    #[arg(short, long, global = true, help = "Disable all logs")]
```

Comment:
Let's tell the parser that this conflicts with `--verbose`:
```suggestion
    #[arg(short, long, global = true, conflicts_with = "verbose", help = "Disable all logs")]
```

Reply:
Added